_data/de.yml

```diff
@@ -0,0 +1,316 @@
+  liGithub: "@bisq-network auf GitHub"
+  liTwitter: "@bisq-network auf Twitter"
+  liYoutube: Bisq YouTube Kanal
+  liSlack: Bisq Slack Arbeitsplatz
+  liForum: Bisq Forum
+  liMailingList: Bisq-Mitarbeiter-Mailingliste 
+  liFreenodeIRC: "#bisq auf Freenode IRC"
+  liSubreddit: r/Bisq Subreddit
+
+main_nav:
+  liDownloads: Downloads
+  liLearnMore: Mehr erfahren
+  liDocumentation: Dokumentation
+  liBlog: Blog
+  liCommunity: Community
+  buttonDownloads: Downloads
+
+os_selector_options:
+  liAllDownloads: Alle Downloads
+  liAllDownloadsText: Zur Downloads Seite
+  liWindows: Für Windows herunterladen
+  liMac: Download für Mac
+  liDebian: Download für Debian/Ubuntu
+  liRedHat: Download für Red Hat/Fedora
+  liArch: Download für Arch Linux
+  liArchText: Arch User Repo
+  liSourceCode: Quellcode
+  liReleaseNotes: Versionshinweise
+  liVerification: Verifizierung
+  liVerificationText1: PGP Signaturen
+  liVerificationText2: PGP Public Key
+
+dao:
+  hLearnMore: "Wollen Sie mehr erfahren?"
+  pLearnMore: "Sie können mehr über die Bisq DAO erfahren, indem Sie die folgenden Ressourcen nutzen, von anspruchsvollen Dokumenten und Videos im ELI5-Stil bis hin zur technischen Spezifikation für Entwickler."
+  conceptualOverview: Konzeptübersicht Dokument
+  pConceptualOverview: "<p class=\"bigger\">Klartext <a href=\"https://docs.bisq.network/user-dao-intro.html\" target=\"_blank\" rel=\"noopener\">Beschreibung</a> der DAO.</p>"
+  bisqDaoBrief: Bisq DAO in kurzen Videos
+  pBisqDaoBrief: "<p class=\"bigger\">Schnelle, kurze <a href=\"https://www.youtube.com/playlist?list=PLFH5SztL5cYPAXWFz-IMB4dBZ0MEZEG_e\" target=\"_blank\">Video-Serie</a> über die Bisq DAO.</p>"
+  technicalOverview: Technischer Überblick - Dokument
+  pTechnicalOverview: "<p class=\"bigger\"><a href=\"https://docs.bisq.network/dao/specification.html\" target=\"_blank\" rel=\"noopener\">Spezifikation</a> der Funktionsweise der Bisq DAO und des BSQ Token.</p>"
+  notConvinced: "Sie sind nicht überzeugt, dass ein Token benötigt wird?"
+  pNotConvinced: "<p class=\"bigger\">Wir haben einen <a href=\"https://bisq.network/blog/bisq-dao-for-bitcoin-maximalists/\" target=\"_blank\">Beitrag</a> speziell für Sie.</p>\n  "
+  seeMore: >
+    <p>Weitere Ressourcen finden Sie in unserem <a href='https://docs.bisq.network/dao.html'>Alles über die Bisq DAO</a> Dokument.</p> <p>Bisherige Mitwirkende finden in <a href='https://docs.bisq.network/compensation.html#get-added-to-genesis'>diesem Link</a> Anweisungen, wie sie der Genesis BSQ-Distribution hinzugefügt werden können.</p>
+downloads:
+  hBisqDownloads: Bisq Downloads
```

Comment:
Need to remove the periods here, otherwise, they result in a double period. Also check pt-PT.yml. I think the others are fine.

The runningOS keys in the `homepage_content` section can have periods, but the runningOS keys in the `downloads` section shouldn't. 

If you want to keep it simple you could just remove all of them...the `homepage_content` doesn't really need periods anyway.

Reply:
Can you show me where the strings for the "running" keys appear on the website?

Reply:
Yeah it's on the Downloads page at the end of the sentence where it says "You appear to be using..."

This is ok, no double period:

![Screenshot from 2019-08-08 04-21-24](https://user-images.githubusercontent.com/735155/62687115-13ccd480-b994-11e9-8dab-9f160dc0479d.png)

This has a double period:

![Screenshot from 2019-08-08 04-20-05](https://user-images.githubusercontent.com/735155/62686987-d49e8380-b993-11e9-85ee-92893ac5cec6.png)

Only a problem for de and pt-pt though...you won't see it for any other language.

Reply:
Oh, thanks. Chrome doesn't display it for some reason here, but it does show up on Firefox.